### ISSUE TITLE
The required namespace "snabbdom" is not available

### ISSUE THREAD

**Issue by BorisKourt** [cjohansen/portfolio#4]
I understand that this is related to: https://github.com/cjohansen/dumdom/issues/23

But I am not sure what the current workaround is. I am using `shadow-cljs` and the rest of the application is in `reagent`. I am assuming that `dumdom` is used for rendering the portfolio UI itself?

**Comment by cjohansen** [comment]
Yes, that's right. I'm working on a fix for this. I'm off for the weekend, but this should be handled early next week at the latest.

**Comment by cjohansen** [comment]
I'm not familiar with shadow-cljs, but if you can try running portfolio from git with the sha `4e25bcda2f8577b6ea863c2af892c7217a3dc94a`, it should hopefully work.

**Comment by BorisKourt** [comment]
I had to use `6beb9521ef5b0f5a631fe6979530f99915730c51` as deps.edn couldn't resolve the sha above.

Encountering new errors in my current project, will try to isolate in a repro.

**Comment by BorisKourt** [comment]
It works, thanks. 

If someone reads this that also has shadow-cljs run separately from the back-end launched via lein: add `classpath:public` to the `:resource-paths` vector of your `project.clj`. And add the dependency to `portfolio` to `project.clj`.

**Comment by cjohansen** [comment]
Feel free to submit a PR on the Readme if there are more ways to configure this 👍
